travel.py

```diff
@@ -77,6 +77,29 @@
                         "example": ["An error message"]
                     }
                 }
+            },
+            "ContextModel": {
+                "type": "array",
+                "items": {"type": "string"}
+            },
+            "ResultDataListModel": {
```

Comment:
I am not too sure about the naming here, 🆘 !

Reply:
I think the name is fine.
But perhaps `ResultList` vs `ResultSingleton` ?